### PR TITLE
CSS styling

### PR DESCRIPTION
**Pull request by stubelshe**
Adding hover states and some formatting.

### CODE REVIEW

File: src/styles/mixins.scss [hunk 1]
```diff
@@ -15,10 +15,10 @@
 }
 
 @mixin transition($transition...) {
-  -moz-transition:    $transition;
-  -o-transition:      $transition;
+  -moz-transition: $transition;
```
Comment (ceanela) [inline src/styles/mixins.scss]: again, please drop these prefixes from the file, they are not needed.

Comment (stubelshe) [reply src/styles/mixins.scss]: Gonna just remove the mixin

**Comment by stubelshe** [comment]
Will update navigation animation post merge.

**Comment by ceanela** [comment]
Looks like your PR is failing in Travis. I'm going to try to get this resolved so your code can be merged.

**Comment by ceanela** [comment]
@stubelshe Forgot to post this, jumping back and forth on a lot of things, but your build is failing because:

screen shot 2017-08-31 at 11 41 39

**Comment by stubelshe** [comment]
@ceanela yep, I knew that would be a problem. Didn't think the build would fail but figured id be fixing it in the next branch. I'll add it back in here to get this done then remove later.

**Comment by stubelshe** [comment]
Ok, mixin added back in

**Pull request merged**